community/plugins/spring-ai-alibaba-starter-plugin-feishu/src/test/java/com/alibaba/cloud/ai/plugin/feishu/SpringAiAlibabaStarterPluginFeishuApplicationTests.java

```diff
@@ -0,0 +1,16 @@
+package com.alibaba.cloud.ai.plugin.feishu;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+/**
+ * @author wudihaoke214
+ * @author <a href="mailto:2897718178@qq.com">wudihaoke214</a>
+ */
+@SpringBootTest
+class SpringAiAlibabaStarterPluginFeiShuApplicationTests {
```

Comment:
If there is no test class, you can delete it

Reply:
ok